src/Components/MovieCard/MovieCard.js

```diff
@@ -1,11 +1,8 @@
 import './MovieCard.css'
 
-function MovieCard({title, img, rating}) {
-
-
-
+function MovieCard({title, img, rating, id, showSingleMovie}) {
   return (
-    <div className='movie-card'>
+    <div className='movie-card' onClick={() => showSingleMovie(id)}>
```

Comment:
I LOVE the refactor here by passing in ID into 'showsinglemove' versus the event that we were passing. Great job!